benchmark_test.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package lru_test
+
+import (
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/lru"
+)
+
+type BenchmarkSuite struct{}
+
+var _ = gc.Suite(&BenchmarkSuite{})
+
+func (*BenchmarkSuite) BenchmarkAddAndEvict1000(c *gc.C) {
```

Comment:
Should we not have a test with something that we can compare it against i.e. slices on addition or BenchmarkAddAndEvict10 vs BenchmarkAddAndEvict100 etc

Reply:
Yeah, I did end up adding a bunch of them. I also added some graphs, etc.